### DISCUSSION TITLE
jvm/class-file-jiegou

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#204] [Announcements]
# jvm/class-file-jiegou

Java程序员进阶之路，小白的零基础Java教程，从入门到进阶，详解Java的类文件（class文件）结构

https://tobebetterjavaer.com/jvm/class-file-jiegou.html

**Comment by liushaowei123** [comment]
怎么通过 索引值，定位到 在class 文件中的位置， 这个是咋算的

**Reply by itwanger** [reply]
我会在新版中增加这块的 QA 说明，敬请期待。

**Comment by soyokra** [comment]
class 文件是一串连续的二进制，由 0 和 1 组成。不是16进制么？

**Reply by soyokra** [reply]
好吧，是编辑器转的

**Comment by chaofengdev** [comment]
`属性表用来存储属性的信息，包括字段的初始值，方法的字节码指令等。`
这里我可以将属性表理解为是对上面字段表、方法表的补充吗？

**Comment by chaofengdev** [comment]
二哥，感觉你这篇写的差点意思哦。
